### PR TITLE
challenge 121 logo code

### PR DESCRIPTION
**Pull request by shiffman**
Adding the code from the Logo challenge (part 1).

### CODE REVIEW

**Comment by shiffman** [comment]
Ooops, yes, thank you @nilsweb!!!!

**Comment by shiffman** [comment]
Let me know if this looks right now!

**Comment by nilsweb** [comment]
Perfect! It works! 🎉 
Just out of curiosity: What should the initial state of the live example be? It's saying `repeat 3 [fd 60 rt 120]` and showing a black canvas... 🤔

File: CodingChallenges/CC_121_Logo/P5/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <title>Logo</title>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/p5.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/addons/p5.dom.min.js"></script>
+  <script src="turtle.js"></script>
+  <script src="sketch.js"></script>
+</head>
+
+<body>
+  <textarea id="code" cols="40" rows="5">
+repeat 3 [fd 60 rt 120]
```
Comment (shiffman) [inline CodingChallenges/CC_121_Logo/P5/index.html]:
```suggestion
fd 60 rt 120 fd 60 rt 120 fd 60 rt 120 
```

**Comment by shiffman** [comment]
Ah, oops! Changing to `fd 60 rt 120 fd 60 rt 120 fd 60 rt 120`

**Comment by shiffman** [comment]
Going to force merge I think the tests failed for something unrelated?

**Comment by shiffman** [comment]
crossing all my fingers and toes. . .

**Comment by nilsweb** [comment]
Seems like CircleCI had some issues pulling the images used for testing... It should work fine!

**Pull request merged**